### DISCUSSION TITLE
Report: OpenVINO not loading networks on an Intel Celeron J4125

### DISCUSSION THREAD

**Discussion by VegetableElk** [immich-app/immich#7273] [General]
Tested on Immich 1.95 and 1.94

The [Celeron J4125](https://www.intel.com/content/www/us/en/products/sku/197305/intel-celeron-processor-j4125-4m-cache-up-to-2-70-ghz/specifications.html) has Intel UHD 600, SSE 4.2 (but *not* AVX) which meets the OpenVINO [support requirements](https://docs.openvino.ai/2023.3/system_requirements.html). (The J4125 and its family is used in multiple Synology NAS variants)

Throws exceptions on network load for both face detection and clip.

```
[E:onnxruntime:, inference_session.cc:1754 operator()] Exception during initialization: /home/onnxruntimedev/onnxruntime/onnxruntime/core/providers/openvino/ov_interface.cc:53 onnxruntime::openvino_ep::OVExeNetwork onnxruntime::openvino_ep::OVCore::LoadNetwork(const string&, std::string&, ov::AnyMap&, std::string) [OpenVINO-EP]  Exception while Loading Network for graph: OpenVINOExecutionProvider_OpenVINO-EP-subgraph_123_0Check 'false' failed at src/inference/src/core.cpp:149:
invalid external data: ExternalDataInfo(data_full_path: 643, offset: 0, data_length: 1806336)
```

Example stack:

![Screenshot 2024-02-20 234856](https://github.com/immich-app/immich/assets/155435730/c3c0444c-08ef-414c-977a-3154a9065448)

**Comment by utrack** [comment]
Same here using Intel N100.

**Comment by spusuf** [comment]
Same on 10600K

**Comment by Mikayex** [comment]
Same here with a Celeron J4105. After updating to 1.96.0, the error changed.

I have this one when doing a search:
![image](https://github.com/immich-app/immich/assets/6607623/93d30061-d8ea-4da7-8655-8b38e0889bd1)

And this one when doing a Smart Search job:
![image](https://github.com/immich-app/immich/assets/6607623/8327b40f-5143-4388-b005-4e114cab06da)

The Face detection seems to work however.

**Comment by VegetableElk** [comment]
Now works on Imich 1.98 🙂
